spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/NoopBindable.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.binding;
+
+import java.util.Set;
+
+/**
+ * A No-op Bindable implementation.
+ *
+ * @author Ilayaperumal Gopinathan
+ */
+public class NoopBindable implements Bindable {
```

Comment:
We should call this BindableAdapter (following the naming patterns of default implementations in Spring projects).